pkg/segment/query/processor/fillnullcommand.go

```diff
@@ -18,16 +18,87 @@
 package processor
 
 import (
+	"io"
+
 	"github.com/siglens/siglens/pkg/segment/query/iqr"
 	"github.com/siglens/siglens/pkg/segment/structs"
+	"github.com/siglens/siglens/pkg/segment/utils"
+	toputils "github.com/siglens/siglens/pkg/utils"
+	log "github.com/sirupsen/logrus"
 )
 
 type fillnullProcessor struct {
 	options *structs.FillNullExpr
 }
 
+func getTheFinalFillValue(fillValue string, qid uint64) (utils.CValueEnclosure, error) {
```

Comment:
In this function, I think you make a DtypeEnclosure just to make a CValueEnclosure, right? Why not make the CValueEnclosure directly? We have a function for this already:
```
func (e *CValueEnclosure) ConvertValue(val interface{}) error {
```